pkg/cmd/auth/logout/logout.go

```diff
@@ -80,7 +80,7 @@ func logoutRun(opts *LogoutOptions) error {
 			hostname = candidates[0]
 		} else {
 			selected, err := opts.Prompter.Select(
-				"What account to you want to log out of?", "", candidates)
+				"What account do you want to log out of?", "", candidates)
```

Comment:
Now that we're here, I think "Which" instead of "What" would be more grammatically correct since there is only a set number of options to choose

Reply:
Actually, let's keep the “Which” change for a separate PR because there's quite a few prompts where this wording could be amended.